essentials.yml

```diff
@@ -1,6 +1,29 @@
 ---
+packaging:
+  config:
+    bundle:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      vendor: "Jenkins project"
+      title: "Jenkins Integration Tests WAR"
+      description: "Jenkins WAR for integration tests"
+    war:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      source:
+        dir: "."
 ath:
   useLocalSnapshots: false
   athRevision: "06caf67d37d945ec5228a80d73714fac692c28d3"
   categories:
     - org.jenkinsci.test.acceptance.junit.SmokeTest
+pct:
+  useLocalSnapshots: false
+  jth:
+    version: 2.38
```

Comment:
Do not override the JTH version. This is at the discretion of each plugin. Sometimes updates are incompatible and require matching test changes. Sometimes a plugin needs to use a newer version of JTH to pick up some new feature.

(If there is some version of JTH which is known to be _too old_ to test against current core, for example because of a plugin split, then you should rather update all the tested plugins to use a _sufficiently new_ version of JTH.)

Reply:
AFAIUI you need at least this JTH version to be able to pass a custom war, so is not a matter of tests to work or not but a precondition of this flow

Reply:
Yes, otherwise JTH does not support custom WAR versions. Obviously we should make it transparent in `essentialsTest()` which should check the minimum required version and bump if needed